include/command_manager.h

```diff
@@ -77,38 +86,50 @@ class CommandManager : public ParamListenerInterface
     control_channel_t * combined;
   } mux_t;
 
-  mux_t muxes[4] = {{&rc_command_.x, &offboard_command_.x, &combined_command_.x},
-                    {&rc_command_.y, &offboard_command_.y, &combined_command_.y},
-                    {&rc_command_.z, &offboard_command_.z, &combined_command_.z},
-                    {&rc_command_.F, &offboard_command_.F, &combined_command_.F}};
+  mux_t muxes[6] = {{&rc_command_.Qx, &offboard_command_.Qx, &combined_command_.Qx},
+                    {&rc_command_.Qy, &offboard_command_.Qy, &combined_command_.Qy},
+                    {&rc_command_.Qz, &offboard_command_.Qz, &combined_command_.Qz},
+                    {&rc_command_.Fx, &offboard_command_.Fx, &combined_command_.Fx},
+                    {&rc_command_.Fy, &offboard_command_.Fy, &combined_command_.Fy},
+                    {&rc_command_.Fz, &offboard_command_.Fz, &combined_command_.Fz}};
 
   // clang-format off
   control_t rc_command_ = {0,
                            {false, ANGLE, 0.0},
                            {false, ANGLE, 0.0},
                            {false, RATE, 0.0},
+                           {false, THROTTLE, 0.0},
+                           {false, THROTTLE, 0.0},
                            {false, THROTTLE, 0.0}};
   control_t offboard_command_ = {0,
                                  {false, ANGLE, 0.0},
                                  {false, ANGLE, 0.0},
                                  {false, RATE, 0.0},
+                                 {false, THROTTLE, 0.0},
```

Comment:
The extra throttles are just the forces commanded in x and y correct? If so is there a way to make this clear? Comment/name change? Or is it clear as you go to implement your own mixer?

Reply:
Coming back after seeing how this is used in the controller this makes more sense. I am still confused why they are all throttles wouldn't only one at a time be throttle? What am I missing?

Reply:
That is a good question. They are all named all `THROTTLE` because they aren't rate or angle commands. These flags are used by the controller to determine if the commands should be multiplied by the `max_throttle` and/or the `max_thrust` parameters.

Since we don't know which one will be used by the controller (X, Y, or Z channel), I assigned them all to be throttle types.

When using the mixer, the constraint is either you use the onboard controller (with throttle settings for all X,Y,Z axes), or you use offboard controller that outputs forces directly (passthrough) to the mixer.